content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -0,0 +1,42 @@
+const React = require("react");
+
+const ActivityFeedItem = React.createClass({
```

Comment:
I probably should've broken up the TopSites component like this too (TopSitesItem). I'll revisit later :+1: